src/Avatar.jsx

```diff
@@ -54,6 +54,14 @@ class Avatar extends React.Component {
 			...other
 		};
 
+		if (this.props.to || this.props.href) {
+			return (
+				<a {...allProps}>
+					{alt}
+				</a>
+			);
```

Comment:
trivial, but this is a lot of line breaks for a small element - unless you expect a lot of embellishment of the `<a>`, I recommend keeping it tight to keep the overall component more readable

```jsx
return <a {...allProps}>{alt}</a>
```